### PR TITLE
Bundle format

### PR DESCRIPTION
**Pull request by arturobrzut**

### CODE REVIEW

File: common/scripts/catalog_build.sh [hunk 1]
```diff
@@ -25,35 +25,28 @@ docker pull "$IMAGE_REPO/$IMAGE_NAME:$MANIFEST_VERSION"
...
 
 VCS_URL=https://github.com/IBM/ibm-common-service-catalog
 VCS_REF=random
 
 echo "Building and pushing catalog"
 docker build -t "$CATALOG_NAME":"$MANIFEST_VERSION" --build-arg \ VCS_REF=${VCS_REF} --build-arg VCS_URL=${VCS_URL} -f Dockerfile .
-docker push "$CATALOG_NAME":"$MANIFEST_VERSION"
-docker tag "$CATALOG_NAME":"$MANIFEST_VERSION" "$CATALOG_NAME":latest
-docker push "$CATALOG_NAME":latest
+# docker push "$CATALOG_NAME":"$MANIFEST_VERSION"
```
Comment (arturobrzut) [inline common/scripts/catalog_build.sh:46]: probably it should not be comment

**Comment by arturobrzut** [comment]
It is ok
we can merge it

**Review by kmigielek — APPROVED** [review]
/lgtm

**Review by kmigielek — APPROVED** [review]
/lgtm

**Pull request merged**